pkg/reconciler/service/controller.go

```diff
@@ -49,12 +49,8 @@ func NewController(
 	configStore := cfgmap.NewStore(logger.Named("config-store"))
 	configStore.WatchConfigs(cmw)
 
-	c := &Reconciler{
-		client:              servingclient.Get(ctx),
-		configurationLister: configurationInformer.Lister(),
-		revisionLister:      revisionInformer.Lister(),
-		routeLister:         routeInformer.Lister(),
-	}
+	c := NewReconciler(servingclient.Get(ctx), configurationInformer.Lister(),
+		revisionInformer.Lister(), routeInformer.Lister())
```

Comment:
Can we format this this way - it's easier to see what's being passed to the constructor

```suggestion
	c := NewReconciler(
		servingclient.Get(ctx), 
		configurationInformer.Lister(),
		revisionInformer.Lister(), 
		routeInformer.Lister(),
	)
```

Reply:
done